CollAction/Startup.cs

```diff
@@ -67,6 +67,23 @@ public void ConfigureServices(IServiceCollection services)
                 options.Password.RequiredLength = 8;
             });
 
+            services.AddAuthentication()
+                    .AddFacebook(options =>
+                    {
+                        options.AppId = Configuration["Authentication:Facebook:AppId"];
```

Comment:
Where are these values coming from? I couldn't find them in the values. Should they be in env_collaction?

Reply:
Also, I think the keys, and the accounts you can use to manage them, should be stored in the password-tool.

Reply:
They are stored in the same place as all the other keys, safely. Either in aws configuration, or the password tool.

But you're right, I should add them to the env_collaction.template as well, for easier configuration.